apps/server/src/shared/domain/entity/task-card.entity.ts

```diff
@@ -22,13 +23,14 @@ export class TaskCard extends BaseEntityWithTimestamps implements ICard, ITaskCa
 
 		this.draggable = props.draggable || true;
 		this.cardType = props.cardType;
-		this.visibleAtDate = props.visibleAtDate;
+
 		this.dueDate = props.dueDate;
 
 		this.cardElements.set(props.cardElements);
 		this.task = props.task;
 		this.cardType = CardType.Task;
 		this.title = props.title;
+		this.visibleAtDate = props.visibleAtDate;
```

Comment:
if its optinal and can be undefined, how is it set here?
also see my notes in `card.types.ts`